packages/core/src/external/carequality/pd/process-inbound-pd.ts

```diff
@@ -2,12 +2,12 @@ import {
   InboundPatientDiscoveryReq,
   InboundPatientDiscoveryResp,
 } from "@metriport/ihe-gateway-sdk";
-import { InboundPatientResource } from "@metriport/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-responses";
+import { PatientResource } from "@metriport/ihe-gateway-sdk";
 import { Address } from "../../../domain/address";
 import { Patient, PatientData } from "../../../domain/patient";
 import { MPI } from "../../../mpi/mpi";
 import { patientMPIToPartialPatient } from "../../../mpi/shared";
-import { toFHIR as convertPatientToFHIR } from "../../fhir/patient";
+import { toIheGatewayPatientResource as convertPatientToFHIR } from "../../fhir/patient";
```

Comment:
we should probably keep the original name to avoid confusion, as there are two diff functions